plugin/trino-base-jdbc/src/test/java/io/trino/plugin/jdbc/TestJdbcMetadataConfig.java

```diff
@@ -32,7 +32,8 @@ public void testDefaults()
                 .setJoinPushdownEnabled(false)
                 .setAggregationPushdownEnabled(true)
                 .setTopNPushdownEnabled(true)
-                .setDomainCompactionThreshold(32));
+                .setDomainCompactionThreshold(32)
+                .setBatchSize(JdbcMetadataConfig.DEFAULT_BATCH_SIZE));
```

Comment:
static import and `DEFAULT_INSERT_BATCH_SIZE`?

Reply:
nit: Use the actual value here instead of a constant.

Reply:
done